src/lib/jsonParseV2/parseNotification.cpp

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
At some point (probably not in parsing logic, but before starting the paring) you shoul check the `Ngsiv2-attrsformat` header (not sure of the name... I'm writting by memory) for `normalized` value, responding with 400 and the proper error message if not satisficed.

Reply:
(Maybe the check is already there but I haven't missed in my review)

Reply:
Fixed in e56e601